server.go

```diff
@@ -408,13 +439,18 @@ func serverWriter(s *Server, w io.Writer, clientAddr string, responsesChan <-cha
 		select {
 		case m = <-responsesChan:
 		default:
+			// Give the last chance for ready goroutines filling responsesChan :)
+			runtime.Gosched()
```

Comment:
@buger this is an interesting part, in their commit message they wrote:

_Increase the probability of message batching on both client and server by calling runtime.Gosched() after requests/responses channels have been drained. This should reduce the number of write() syscalls (especially on go1.5)_

And, in our customer's profiles we saw a high mem usage on writtes... do you think would worth to add this to the patch?

Reply:
Interesting. For sure lets add it.